src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -708,7 +710,7 @@ public void actionPerformed(ActionEvent e) {
             }
         });
         mainTable.getActionMap().put(Actions.COPY, new AbstractAction() {
-
+        
```

Comment:
Please double check your IDE config. We recommend IntelliJ. See https://github.com/jabref/jabref/wiki/Guidelines-for-setting-up-a-local-workspace for detailed steps. Please let @Siedlerchr know if you use Eclipse.

Reason: We do not want to have code changes only because of a style.

Reply:
I am using IntelliJ IDE and followed the IntelliJ IDEA Code Style Configuration  (https://github.com/JabRef/jabref/tree/master/config) before posting any commits to this project.

![config](https://user-images.githubusercontent.com/8946793/66707305-c0ce1f80-ed3e-11e9-94f7-b144f66a97c9.JPG)

Therefore the malformed code style has been in the file before I made any changed and was corrected automatically by the IDE.

Reply:
Interesting. Did you format all code? - At the diff of all the files (button on GitHub) there were spaces introduced to an empty line.

Reply:
Yes, I always format the whole file after any changes (for example to remove unused imports). But actually I noticed something odd going on here. Eventhough I had the JabRef code style configured, it was still overridden by the default editor config (for some reason). This should be fixed by now and we got proper formatting as desired.